app/views/exam_templates/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for :title, ExamTemplate.model_name.human.pluralize %>
+
+<% content_for :content do %>
```

Comment:
Good - so you don't actually need the outer `content_for`; the contents of the view is actually automatically put in `content` in the layout.